opentelemetry-tracing/opentelemetry-java-tracing/okhttp-and-jedis/README.md

```diff
@@ -0,0 +1,44 @@
+# Java Agent
+
+This example shows the use of the [Splunk distribution of OpenTelemetry Java instrumentation project](https://github.com/signalfx/splunk-otel-java).  
+The Java agent performs bytecode manipulation of the target application using the [JVM
+Instrumentation](https://docs.oracle.com/javase/8/docs/api/java/lang/instrument/package-summary.html)
+interface and enables tracing of many popular libraries and frameworks without
+modification of the target application's source code.
+
+Here are some environment variables you might need to set to configure the Java
+agent:
+```
+$ # This is the ZIPKIN exporter - compatible with SignalFX SmartAgent
+$ export OTEL_EXPORTER=zipkin
+
+$ # This is the default endpoint url
+$ export OTEL_ZIPKIN_ENDPOINT=http://localhost:9080/v1/trace
+
+$ # Replace this with your application's common name
+$ export OTEL_ZIPKIN_SERVICE_NAME=unnamed-java-app
+```
+
+To obtain the latest Java agent, download it to your host's filesystem:
+
+```
+curl -sSL -o /opt/opentelemetry-javaagent-all.jar 'https://github.com/signalfx/signalfx-otel-java/releases/latest/download/signalfx-otel-javaagent-all.jar'
```

Comment:
This will not until repo is made public

Reply:
Yes, I understand we want to release these examples along with our distro of the OTEL agent (correct me if I'm wrong!).